diff --git a/dist/index.mjs b/dist/index.mjs
index 2610829d30fba8df7d722d21e33fb710d770f46b..05e827ecf7c778a89b87317e244edb7fd843700e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -123,8 +123,6 @@ function relativeMinimatch(pattern, relativePath, cwd) {
   const isParent = relativePath.startsWith("..");
   if (!isParent)
     return `${negated}${relativePath}${cleanPattern}`;
-  if (!RE_PARENT_PATH.test(relativePath))
-    throw new Error("The ignore file location should be either a parent or child directory");
   if (cleanPattern.startsWith("**"))
     return pattern;
   const parents = relative(resolve(cwd, relativePath), cwd).split(RE_PATH_SEP);
